schemas/stsci.edu/asdf/0.1.0/transform/regions_selector.yaml

```diff
@@ -0,0 +1,55 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/transform/regions_selector"
+tag: "tag:stsci.edu:asdf/0.1.0/transform/regions_selector"
+title: >
+  Represents a RegionsSelector transform.
+description: |
+  A model which maps regions to their corresponding transforms.
+
+allOf:
+  - $ref: transform
+  - type: object
+    properties:
+      mask:
+        description: |
+          An array with the shape of the detector/observation.
+          Pixel values are of type int or str and represent
+          region labels as in the RegionSelector transform.
+          Pixels which are not within any region have value 0 or " ".
+
+        $ref: ./selector_mask
```

Comment:
I see.  But as it's defined now, one has to do:

```
transform: !regions_selector
   mask:
      mask: [...array...]
      inverse:
          mask: [...array...]
```

which seems confusing to me.

Maybe have `mask` and `inverse_mask`, both of which are arrays?